### PR TITLE
reworked design

### PR DESCRIPTION
**Pull request by MyNameJeremy**

### CODE REVIEW

File: admin_panel/style.css [hunk 1]
```diff
@@ -9,80 +9,104 @@
...
-  padding-left: 32px;
-  text-align: start;
-  border-right: var(--border-style);
-  border-left: var(--border-style);
+tbody {
+  color: #555555;
 }
-td:first-child {
-  border-left: 0;
+table tbody tr {
```
Comment (peter-schweitzer) [inline admin_panel/style.css]: remove unnecessary specifiers `table tbody`

File: admin_panel/style.css [hunk 2]
```diff
@@ -9,80 +9,104 @@
...
+  height: 50px;
 }
-td:last-child {
-  border-right: 0;
+table tbody tr:last-child {
```
Comment (peter-schweitzer) [inline admin_panel/style.css]: remove unnecessary specifiers `table`

File: admin_panel/style.css [hunk 3]
```diff
@@ -9,80 +9,104 @@
...
+  justify-content: center;
+  flex-wrap: wrap;
 }
 
-table#kv-table {
+table {
+  max-width: 50%;
+  border-spacing: 1;
   border-collapse: collapse;
+  background: white;
```
Comment (peter-schweitzer) [inline admin_panel/style.css]: stay consistent use `#fff` (or `#ffffff`)

File: admin_panel/style.css [hunk 4]
```diff
@@ -9,80 +9,104 @@
...
+  background: #36304a;
 }
-
-td {
-  padding-left: 32px;
-  text-align: start;
-  border-right: var(--border-style);
-  border-left: var(--border-style);
+tbody {
+  color: #555555;
```
Comment (peter-schweitzer) [inline admin_panel/style.css]: stay consistent use `#555`

File: admin_panel/style.css [hunk 5]
```diff
@@ -9,80 +9,104 @@
...
-  height: 16px;
-  border-bottom: var(--border-style);
+table * {
+  position: relative;
 }
-
-tr:first-child {
-  border-bottom: 0;
+table td,
+table th {
```
Comment (peter-schweitzer) [inline admin_panel/style.css]: remove unnecessary specifiers `table`

File: admin_panel/style.css [hunk 6]
```diff
@@ -9,80 +9,104 @@
...
-  border-bottom: 0;
+thead {
+  color: #fff;
 }
-
-th {
-  width: 50%;
-  height: 128px;
-  text-align: center;
+table thead tr {
```
Comment (peter-schweitzer) [inline admin_panel/style.css]: remove unnecessary specifiers `table`

File: admin_panel/style.css [hunk 7]
```diff
@@ -9,80 +9,104 @@
...
-tr {
-  height: 16px;
-  border-bottom: var(--border-style);
+table * {
+  position: relative;
 }
-
-tr:first-child {
-  border-bottom: 0;
+table td,
```
Comment (peter-schweitzer) [inline admin_panel/style.css]: missing \n (first occurrence)

**Review by MyNameJeremy — COMMENTED** [review]
resolved comments

File: admin_panel/style.css [hunk 8]
```diff
@@ -79,24 +79,30 @@ table {
...
+th {
   padding-left: 15px;
 }
+
 thead {
   color: #fff;
 }
-table thead tr {
+
+thead tr {
```
Comment (peter-schweitzer) [inline admin_panel/style.css]: change to `thead>tr`

**Review by MyNameJeremy — COMMENTED** [review]
resolved.

**Review by peter-schweitzer — APPROVED** [review]

**Pull request merged**